metadata.rb

```diff
@@ -8,7 +8,7 @@
 license          'Apache-2.0'
 description      'Installs/Configures Visual Studio'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '2.0.0'
+version          '1.2.6'
```

Comment:
Did this fall back a version?

Reply:
That was actually me accidentally copying full metadata from upstream without noticing the major version bump
It was never intended to be on v2 in this fork anyway